aries_cloudagent/protocols/didexchange/v1_0/manager.py

```diff
@@ -702,13 +702,22 @@ async def accept_response(
         conn_rec = None
         if response._thread:
             # identify the request by the thread ID
-            try:
-                async with self.profile.session() as session:
+            async with self.profile.session() as session:
+                try:
                     conn_rec = await ConnRecord.retrieve_by_request_id(
-                        session, response._thread_id
+                        session, response._thread_id,
+                        their_role=ConnRecord.Role.RESPONDER.rfc23,
                     )
-            except StorageNotFoundError:
-                pass
+                except StorageNotFoundError:
+                    pass
+                if not conn_rec:
+                    try:
+                        conn_rec = await ConnRecord.retrieve_by_request_id(
+                            session, response._thread_id,
+                            their_role=ConnRecord.Role.RESPONDER.rfc160,
```

Comment:
It looks like the rfc23 roles should be used, but the created connection records are using the rfc160 roles ... so I included checks for both